src/chapters/08/main.tex

```diff
@@ -115,62 +115,67 @@ \section{Theory}\label{sec:theory}
-scheduled for a different day, thus $|T| = |M| = 14$ in this case.
-Let $\{X_{mt} \text{ for } m \in M \text{ and } t \in T\}$ be a set of binary
-decision variables, that is $X_{mt} = 1$ if module $m$ is scheduled for time
-$t$, and $0$ otherwise.
+scheduled for a different day, thus \(|T| = |M| = 14\) in this case.
+Let \(\{X_{mt} \text{ for } m \in M \text{ and } t \in T\}\) be a set of binary
+decision variables, that is \(X_{mt} = 1\) if module \(m\) is scheduled for time
+\(t\), and \(0\) otherwise.
 
 There are six distinct sets of modules in which exams cannot be scheduled
-simultaneously: $A_c$, $A_o$ representing core and optional art modules
-respectively; $B_c$, $B_o$ representing core and optional biology modules
-respectively; and $C_c$, $C_o$ representing core and optional chemistry modules
+simultaneously: \(A_c\), \(A_o\) representing core and optional art modules
+respectively; \(B_c\), \(B_o\) representing core and optional biology modules
+respectively; and \(C_c\), \(C_o\) representing core and optional chemistry modules
 respectively.
-Therefore $M = A_c \cup A_o \cup B_c \cup B_o \cup C_c \cup C_o$.
+Therefore \(M = A_c \cup A_o \cup B_c \cup B_o \cup C_c \cup C_o\).
 
 Additionally there are further clashes between these sets:
 \begin{itemize}
-  \item No modules in $A_c \cup A_o$ can be scheduled together as they may
+  \item No modules in \(A_c \cup A_o\) can be scheduled together as they may
   share students, this is given by the constraint in inequality~\ref{eqn:clique1}.
-  \item No modules in $B_c \cup B_o \cup A_c$, can be scheduled together as
+  \item No modules in \(B_c \cup B_o \cup A_c\), can be scheduled together as
   they may share students, given by inequality~\ref{eqn:clique2}.
-  \item No modules in $B_c \cup B_o \cup C_o$, can be scheduled together as
+  \item No modules in \(B_c \cup B_o \cup C_o\), can be scheduled together as
   they may share students, given by inequality~\ref{eqn:clique3}.
-  \item No modules in $B_o \cup C_c \cup C_o$, can be scheduled together as
+  \item No modules in \(B_o \cup C_c \cup C_o\), can be scheduled together as
   they may share students, given by inequality~\ref{eqn:clique4}.
 \end{itemize}
 
-Let's also define $\{Y_t \text{ for } t \in T\}$ as a set of auxiliary binary
-decision variables, where $Y_t$ is 1 if time slot $t$ is being used. This is
+Define \(\{Y_t \text{ for } t \in T\}\) as a set of auxiliary binary
+decision variables, where \(Y_t\) is 1 if time slot \(t\) is being used. This is
 enforced by Inequality~\ref{eqn:auxiliary}.
 
-Finally we have one final constraint, Equation~\ref{eqn:schedule_all_modules},
-which ensures all modules are scheduled once and once only.
-Thus altogether our integer program becomes:
+Equation~\ref{eqn:schedule_all_modules},
+ensures all modules are scheduled once and once only.
+Thus altogether the integer program becomes:
+% TODO Capitalize Integer Program or not? (VK I don't care, but need to check
+% for consistency)
```

Comment:
I don't like capitalising things that don't need it.

Reply:
Fine by me, I think I only did it because you had capitalised some other stuff (but maybe that was me in the first place 😆 ).